client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -345,6 +347,39 @@ export default React.createClass( {
 				section="domain-management" />;
 	},
 
+	pendingTransfer() {
+		const domains = this.getDomains().filter( domain => domain.pendingTransfer );
+
+		if ( domains.length !== 1 ) {
```

Comment:
While probably not the most common case, but what if the user has more than one pending transfer domains? I can imagine a user that has two custom domains decides to move from wpcom somewhere else and initiates transfers of both domains. And, boom, no notice ;)